src/nmap.js

```diff
@@ -52,6 +52,9 @@ function transform(hosts) {
                     serviceProduct: openPort.serviceProduct || null,
                     serviceVersion: openPort.serviceVersion || null,
                     scripts: openPort.scriptOutputs || null,
+                    ostype: openPort.ostype || null,
+                    conf: openPort.conf || null,
```

Comment:
A full name seems to be more readable in y opinion like "confidence" instead of "conf"

see: https://nmap.org/book/output-formats-xml-output.html

> The conf attribute measures the confidence Nmap has that the service name is correct. The values range from one (least confident) to ten.